common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -65,10 +68,74 @@ innovation_small_soldiers = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
 #culture_group_regional
 ####################
 ##Tribal##
 ####################
+innovation_waterspeaking = {
```

Comment:
Jinyu will have it even worshipping Ragnaros and having Water magic banned.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110717107-e0ae8f00-8221-11eb-93d4-25c19bd61cf9.png)

</details>

Reply:
Could I change the name to "Nature Attuned", would that work?

Reply:
> Could I change the name to "Nature Attuned", would that work?

All kinds of magic can be banned. Nature, all elements.